babel/localtime/_helpers.py

```diff
@@ -0,0 +1,43 @@
+try:
+    import zoneinfo
+except ModuleNotFoundError:
+    zoneinfo = None
+    import pytz
```

Comment:
Same thing here, let's support both:
```suggestion
try:
    import pytz
except ModuleNotFoundError:
    pytz = None

try:
    import zoneinfo
except ModuleNotFoundError:
    zoneinfo = None
```